examples/spiders/recaptcha.py

```diff
@@ -0,0 +1,33 @@
+import scrapy
+from twisted.python.failure import Failure
+
+from scrapypuppeteer import PuppeteerRequest
+from scrapypuppeteer.actions import GoTo, RecaptchaSolver, Click
+from scrapypuppeteer.response import PuppeteerResponse
+
+
+class RecaptchaSpider(scrapy.Spider):
+    name = "recaptcha"
+
+    start_urls = ["https://www.google.com/recaptcha/api2/demo"]
+
+    def start_requests(self):
+        for url in self.start_urls:
+            action = GoTo(url=url)
+            yield PuppeteerRequest(action=action, callback=self.solve_recaptcha, errback=self.error, close_page=False)
+
+    def solve_recaptcha(self, response: PuppeteerResponse, **kwargs):
```

Comment:
As far as i can understand this code shoul be removed, so we could solve such captcha's automatically.